src/plugins/intel_cpu/tests/unit/transformations/arm/snipptes_mark_skipped.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <gtest/gtest.h>
+#include <subgraph_customizable.hpp>
+#include <snippets_helpers.hpp>
+#include <transformations/snippets/aarch64/pass/snippets_mark_skipped.hpp>
+#include "snippets/pass/tokenization.hpp"
+#include "snippets/pass/collapse_subgraph.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class SnippetsMarkSkippedTests : public TransformationTestsF {
+public:
+    void run() {
+        ASSERT_TRUE(model);
+        ov::snippets::pass::SnippetsTokenization::Config config = get_default_config();
+        manager.register_pass<ov::intel_cpu::SnippetsMarkSkipped>();
+        manager.register_pass<ov::snippets::pass::EnumerateNodes>();
+        manager.register_pass<ov::snippets::pass::TokenizeSnippets>(config);
+    }
+};
+
+TEST_F(SnippetsMarkSkippedTests, smoke_Snippets_SkipConvFused_ConvMulRelu) {
+    std::vector<std::shared_ptr<Node>> eltwiseOps {std::make_shared<ov::op::v1::Multiply>(),
+                                                   std::make_shared<ov::op::v0::Relu>(),
+                                                   std::make_shared<ov::op::v0::Relu>()};
+    std::vector<PartialShape> inputShapes {{1, 2, 16, 16}, {1, 2, 1, 16}};
+    const auto &f = ConvMulActivationFunction(inputShapes, eltwiseOps);
+    model = f.getOriginal();
+    // Fully tokenizable, since Mul isn't fused into Convolution
+    model_ref = f.getReference();
+    run();
+}
+
+TEST_F(SnippetsMarkSkippedTests, smoke_Snippets_SkipConvFused_ConvSumRelu) {
```

Comment:
Cool! We have some tests for SnippetsMarkSkipped on arm!

Could you please also add tests with `Bias`?
For example:
1. Original model: `Conv->Bias->Relu`.
Ref: `Conv->Bias->Relu` (as original since `SnippetsMarkSKipped` should mark them as `Fused` all of them)
2. (if defined CPU_WITH_ACL)
Original model: `Conv->Bias->Relu->Relu`.
Ref: `Conv->Bias->Relu->Subgraph(Relu)` - since ACL Conv supports only one bias.
3. Original model: `MatMul->Bias->Relu->Relu`.
Ref: as original
4. Original model: `MatMul->Bias->Relu->Div`.
Ref: `MatMul->Bias->Relu->Subgraph(Div)`.

Reply:
Applied. Thanks Alexandra!